src/Migrations/Version20250520161231.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use MatchBot\Application\Environment;
+
+final class Version20250520161231 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'BG2-2890: Delete unwanted donor account';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql(
+            <<<SQL
+            DELETE FROM DonorAccount WHERE matchbot.DonorAccount.uuid = '1ed6fdf0-71d6-6ab0-98a0-11455be3c07b' LIMIT 1
+            SQL
+
+        );
+
+        if (Environment::current() === Environment::Staging) {
+            return; // in case of collisions between donation ids we're wanting to adjust in prod and any in staging.
+        }
+
+        $this->addSql(
+            <<<SQL
+            UPDATE Donation SET 
+                                Donation.donorEmailAddress = 'email-redacted',
```

Comment:
```suggestion
                               donorEmailAddress = 'email-redacted',
```
When I simulate the update in pMA it seems to be unable to work out which field this is without this change. Not totally sure why but seems safer to do this just in case that's accurate.